packages/valory/skills/dynamic_nft_abci/skill.yaml

```diff
@@ -84,6 +83,9 @@ models:
   http_dialogues:
     args: {}
     class_name: HttpDialogues
+  ipfs_dialogues:
+    args: {}
+    class_name: IpfsDialogues
```

Comment:
IpfsHandlers are also required.

Reply:
Thanks! I was also missing a couple of configs in the chained skill. Fixed now.